### ISSUE TITLE
standalone_security_server_tutorial.md

### ISSUE THREAD

**Issue by RamSailopal** [digitaliceland/Straumurinn#6]
The information in the guide doesn't seem to be correct.

Seems like the http proxy port is actually port 8080 and not 80?

Also, I have followed the guide and tried to set up the standalone security server in one Docker container (called secserv) with the ping-pong api endpoint served from a second container (pingserv) and the X-client running in a third container (xclient). All containers are named in such a way that they can resolve to one another via container name (this has been verified in each container)

The guide has been followed step by step but when it comes to querying the API through the security server, the following message appears:

     {"type":"Server.ClientProxy.NetworkError","message":"ss: Temporary failure in name resolution","detail":"32b18e23-c256-4824-830a-7af22a5b9f41"}

I then setup an additional REST service (TEST124) through the security service UI and this service references an external API:

**https://api.hel.fi/kore/v1/**

The setup has been tested via the following curl api call:

     curl -l -X GET 'http://secserv:8080/r1/CS/ORG/1111/TestService/TEST124/schools' -H 'accept: application/json' -H 'X-Road-Client:CS/ORG/1111/TestClient'

The same error message appears yet again:

     {"type":"Server.ClientProxy.NetworkError","message":"ss","detail":"9c89bd10-4004-4d16-88f8-dca36671679d"}

The API domain name **api.hel.fi** can be resolved in all containers and so I am not sure where the issue lies.

**Comment by arejae** [comment]
based on my test, port 80 is correct.

problem is the OCSP response. both Auth certificate and Sign Certificate status is Unknown. 

I think the missing part on this tutorial are :-  certificate authority, timestamping and ocsp service.

**Comment by Christopher-Moreira** [comment]
No, the actual problem is the port. You should get your docker to run on :8080 and in the final command you should also run the :8080

./rest-client -cmd ping -service CS/ORG/1111/TestService/TEST123 -client CS/ORG/1111/TestClient -ss http://localhost:8080  <--- the port should be in the same as Docker to pull the pong request.
The certificate error only happen in the 6.0.0 version, in the newer ones the proxy is fixed, but the port's not.

<img width="535" height="1039" alt="Image" src="https://github.com/user-attachments/assets/48cf6ad9-6de3-4f8a-a06a-a29d175a3796" />
